src/box/lua/config/instance_config.lua

```diff
@@ -1,6 +1,5 @@
 local schema = require('experimental.config.utils.schema')
 local tarantool = require('tarantool')
-local compat = require('compat')
```

Comment:
> NO_CHANGELOG=bugfix

We have 'Bugs fixed' section in the release notes and this particular change is a user-visible one. I propose to add a changelog entry.

From a user perspective it looks like the `box_cfg_replication_sync_timeout` = `old` has no effect when a tarantool instance is started from a YAML configuration.

Reply:
Added.